### ISSUE TITLE
[steamshovel] hese-style screenshots (Trac #1071)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1065]
Make it easy to generate hese-style screenshots, with colorbar for time.  Like:
http://gallery.icecube.wisc.edu/web/var/resizes/Press-Root-Album/Events/Event_stills3.png

Migrated from https://code.icecube.wisc.edu/ticket/1071
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:58", 
    "description": "Make it easy to generate hese-style screenshots, with colorbar for time.  Like:\nhttp://gallery.icecube.wisc.edu/web/var/resizes/Press-Root-Album/Events/Event_stills3.png", 
    "reporter": "david.schultz", 
    "cc": "hdembinski", 
    "resolution": "fixed", 
    "_ts": "1550067178841456", 
    "component": "combo core", 
    "summary": "[steamshovel] hese-style screenshots", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-07-28T19:28:36", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150728T19:28:47: david.schultz changed attachment from "" to "hese_screenshot.py"

**Comment by icecube-trac** [comment]
Trac update at 20150801T12:41:53: 
* hdembinski changed priority from "blocker" to "major"
* hdembinski commented:

> This is not a blocker.

**Comment by icecube-trac** [comment]
Trac update at 20150801T12:43:19: hdembinski commented:

> I need more information. Do they want a set of artists that precisely reproduces this screenshot?

**Comment by icecube-trac** [comment]
Trac update at 20150828T16:41:20: 
* david.schultz commented:

> my latest version of the screenshot script, with viewing angles from cweaver


* david.schultz changed attachment from "" to "hese_screenshot.2.py"

**Comment by icecube-trac** [comment]
Trac update at 20151117T22:23:43: david.schultz changed milestone from "offline V15-11-00" to "January 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160222T16:33:11: david.schultz changed milestone from "February 2016" to "April 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160622T18:17:46: 
* david.schultz changed priority from "major" to "critical"
* david.schultz changed status from "new" to "accepted"
* david.schultz commented:

> The WIPAC interns want to use this script with the steamshovel app. I guess that raises the importance, since they'd like it on a timescale of days.
> 
> The general design would be a menu option like the screenshot (or maybe just a check box inside there?).
> 
> I'll try to make this work, though I'm not sure on some of the Qt stuff.

**Comment by icecube-trac** [comment]
Trac update at 20160627T19:12:42: 
* david.schultz commented:

> In changeset 147871:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="147871"
> hese screenshots! not perfect, but good enough for now. closes #1065
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:58: 
* olivas commented:

> Milestone May 2016 deleted


* olivas changed milestone from "May 2016" to ""
